python/eos/analysis.py

```diff
@@ -724,9 +726,16 @@ def sample_nested(self, bound='multi', nlive=250, dlogz=1.0, maxiter=None, seed=
         .. note::
            This method requires the dynesty python module, which can be installed from PyPI.
         """
-        import dynesty
+        import dynesty, tqdm
+        from functools import partial
+
+        if print_function is None:
+            print_function = partial(dynesty.results.print_fn, pbar=tqdm.tqdm())
+
         sampler = dynesty.DynamicNestedSampler(self.log_likelihood, self._prior_transform, len(self.varied_parameters), bound=bound, nlive=nlive, rstate = np.random.Generator(np.random.MT19937(seed)), sample=sample)
-        sampler.run_nested(dlogz_init=dlogz, maxiter=maxiter, print_progress=print_progress)
+        sampler.run_nested(dlogz_init=dlogz, maxiter=maxiter, print_progress=print_progress, print_func=print_function)
+        while sampler.results['niter'] < miniter:
+            sampler.run_nested(dlogz_init=dlogz, maxiter=maxiter, print_progress=print_progress, print_func=print_function)
```

Comment:
Uff, you are running the sampler from scratch! Why don't you use `sampler.add_batch(mode='full')` instead?

Reply:
Ah, I had not seen the ``full`` mode documentation in the dynesty API. Switched!